src/Faker/Provider/pt_BR/LocalUtils.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Faker\Provider\pt_BR;
+
+class LocalUtils
+{
+    /**
+     * Calculates one MOD 11 check digit based on customary Brazilian algorithms.
+     * @link http://en.wikipedia.org/wiki/Check_digit
+     * @link http://pt.wikipedia.org/wiki/CNPJ#Algoritmo_de_Valida.C3.A7.C3.A3o
+     * @link http://en.wikipedia.org/wiki/Cadastro_de_Pessoas_F%C3%ADsicas#Validation
+     *
+     * @param string $numbers Numbers on which generate the check digit
+     * @return integer
+     */
+    public static function checkDigit($numbers)
```

Comment:
@fzaninotto - I've kept name of function as it was before. If it's ok to change name of it, I'll do it too. Class was created just to remove require inside Providers and let it be done via autoloader